### PR TITLE
Changed name of CLI parameter '-mutexPolicy' to '-reqMutexPolicy'

### PR DESCRIPTION
**Pull request by kzangeli**
That and a functest changed, of course :)

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -8,3 +8,4 @@
 - Fix: updateContext propagated to the entire Service Path subtree, instead of just scoping to the particular Service Path in the Fiware-Service header (Issue #885)
 - Fix: Mongo driver migrated to legacy-1.0.2
 - Add: Database connection pool for mongo. Default size of the pool is 10 connections. This is changed using the CLI parameter -dbPoolSize (Issue #909)
+- Fix: Changed name of CLI parameter '-mutexPolicy' to '-reqMutexPolicy' (No issue)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: I would suggest not using this line and just adjunting the one some lines above to:

```
- Add: New CLI parameter for the mutex policy: -reqMutexPolicy (Issue #910)
```

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Ah, did catch that one ... Of course!

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Fixed in b0c2242

**Comment by fgalan** [comment]
LGTM

**Pull request merged**